### PR TITLE
Upstream mods

### PR DESCRIPTION
**Pull request by nxg**
Three minor changes, of points which I found it useful to make in a presentation of this material (one partly duplicates an upstream fix).

Apologies: I read the remark about _not_ committing generated `.html` files only after I'd committed the first two of these – I hope that doesn't cause trouble.

### CODE REVIEW

**Comment by nxg** [comment]
Doh! Not a terribly helpful title for the pull request...

**Comment by shwina** [comment]
:D Was a bit confused about what this PR is supposed to be. Thanks! I'll review soon

**Comment by nxg** [comment]
Excellent.

I've done only a couple of pull requests on github, so if you've any suggestions on the style or content of the patches, I'm very happy to be taught!

File: 03-pipefilter.md [hunk 1]
```diff
@@ -154,6 +154,17 @@ $ cat lengths.txt
...
 ~~~
 
+> ## Output page by page {.callout}
+>
+> We'll continue to use `cat` in this lesson, for convenience and consistency,
+> but it has the disadvantage that,
+> if a file is long, you get *all* of it dumped onto your screen.
+> Probably more useful in practice is the command `less`,
+> as in `$ less lengths.txt`,
+> which displays a screenful of the file, and then stops.
```
Comment (shwina) [inline 03-pipefilter.md]: I get the feeling that there are too many commas in the above two sentences. Maybe split into smaller sentences? Sorry for the pedantic comment!

**Comment by shwina** [comment]
Thanks - looks great to me!

**Comment by shwina** [comment]
@nxg - would you please make the changes mentioned?

**Comment by nxg** [comment]
I think I've done this (I think I was at some level putting off doing this because of the joy of wrestling with git).  Github suggests you can see the changes – is that true?

**Comment by shwina** [comment]
Now I can! Thanks so much. I'll merge

**Comment by nxg** [comment]
Thanks!

**Pull request merged**